client/src/components/form/EducationInfo.tsx

```
@@ -41,20 +41,19 @@ function AcademicDetails({
       ...formData,
       [section]: {
         ...formData[section],
-        [field]: field === 'passingYear' ? parseInt(value) : value,
+        [field]: field === 'passingYear' ? parseInt(value, 10) : value,
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)